client/src/app/submission/submit/submit.component.html

```diff
@@ -46,11 +46,18 @@ <h3 class="vf-text vf-text-heading--4">What's next?</h3>
     </mat-checkbox>
 
   </div>
+  <div>
+    <mat-checkbox
+      [(ngModel)]="submitMetadataToDcp" labelPosition="after"
+      (change)="onSubmitMetadataToDspChange()">
+      Submit only metadata to the <a href="https://data.humancellatlas.org" class="vf-link">Human Cell Atlas</a>.
```

Comment:
Is this url same for dev/staging/prod?

Reply:
yes it is, it follows the existing URL that is used here: https://github.com/ebi-ait/ingest-ui/blob/d705eda55cefe7d04e41c544906107696c073896/client/src/app/submission/submit/submit.component.html#L60